README.md

```diff
@@ -12,6 +12,8 @@ Run `make`.
 
 Run `make test` to run all the tests.
 
+haris: why don't we create the test data (i.e.) ourselves?
```

Comment:
If we create a sheet by ourselves, that will be stored in the user's root drive folder and there is no option to create a spreadsheet in the specified folder from API.
Creating the spreadsheet programmatically will add extra logic to our code that is irrelevant to the connector.
Not sure but, There can also be some permission issues if the user keeps the scope of the credential limited.
It is easier to provide the spreadsheet and sheet id manually.

Reply:
The acceptance tests are run automatically (e.g. in a PR) or by a developer, so the account being used is actually a test account, no?

If that is true, I see no issues creating a sheet in the acceptance test. It would also not be part of the connector itself.

The issues I see with manually providing a spreadsheet and a sheet ID are: it's manual and no guarantee that a test will start with a clean slate (e.g. if a cleanup failed).

Reply:
Currently, I have set up the sheet with my personal account and added a separate sheet specifically for automation tests.
Any new contributor will like to run the tests in their local as well. I personally will prefer no sheets be added to my drive.

Reply:
We can create a dedicated account for our Github actions. But, then we are also expecting all the contributors to have a dedicated account for the development purpose.

Reply:
That's true, which is probably acceptable. But that might be a matter of personal preference. I'm mark this as resolved, as I don't think what I'm suggesting in super big improvement.: )